templates/Cargo-manifest.toml

```diff
@@ -8,4 +8,3 @@ strip = true
 opt-level = 's'
 lto = false
 codegen-units = 1
-panic = 'abort'
```

Comment:
Why remove `panic = 'abort'`?

Reply:
It doesn't seem to make any difference. My guess is that the custom panic handler is used instead of either abort or unwind.

Reply:
I'm not really sure. I'll add it back.

Reply:
See https://docs.rust-embedded.org/book/start/panicking.html

> In the standard library panicking has a defined behavior: it unwinds the stack of the panicking thread, unless the user opted for aborting the program on panics.

> In programs without standard library, however, the panicking behavior is left undefined. A behavior can be chosen by declaring a #[panic_handler] function. This function must appear exactly once in the dependency graph of a program, and must have the following signature: fn(&PanicInfo) -> !, where [PanicInfo](https://doc.rust-lang.org/core/panic/struct.PanicInfo.html) is a struct containing information about the location of the panic.

This seems to indicate that `panic = 'abort'` doesn't really have any effect on no_std targets.

Reply:
Or it might be the riscv64imac-unknown-none-elf target already defines `"panic-strategy": "abort"`:

```
$ rustc +nightly -Z unstable-options --print target-spec-json --target riscv64imac-unknown-none-elf
{
  "arch": "riscv64",
  "code-model": "medium",
  "cpu": "generic-rv64",
  "data-layout": "e-m:e-p:64:64-i64:64-i128:128-n32:64-S128",
  "eh-frame-header": false,
  "emit-debug-gdb-scripts": false,
  "features": "+m,+a,+c",
  "is-builtin": true,
  "linker": "rust-lld",
  "linker-flavor": "ld.lld",
  "llvm-target": "riscv64",
  "max-atomic-width": 64,
  "panic-strategy": "abort",
  "relocation-model": "static",
  "supported-sanitizers": [
    "kernel-address"
  ],
  "target-pointer-width": "64"
}
```